live_pipeline/load/test_database_functions.py

```diff
@@ -1,7 +1,87 @@
-'''libraries required to test database functions'''
-import pytest
-from database functions import add_cycle_information, add_botanist_information, add_species_information, add_plant_information
+"""libraries required to test database functions"""
```

Comment:
Would be tempted to rename to something like "Unit tests to be run with pytest for the database functions."